alpine-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -21,9 +16,9 @@ http {
   tcp_nopush  on;
   tcp_nodelay on;
 
-  keepalive_timeout 90;
+  keepalive_timeout 300;
```

Comment:
I was trying to update my nginx config to match the current best nginx config I have from other projects.